resolver/resolver.cc

```diff
@@ -450,7 +450,7 @@ class ResolveConstantsWalk {
         auto uaSym = ctx.state.enterMethodSymbol(core::Loc::none(), item.klass, core::Names::unresolvedAncestors());
         core::TypePtr resultType = uaSym.data(ctx)->resultType;
         if (!resultType) {
-            uaSym.data(ctx)->resultType = core::TupleType::build(ctx, {ancestorType});
+            uaSym.data(ctx)->resultType = core::make_type<core::TupleType>(vector<core::TypePtr>{ancestorType});
```

Comment:
Do you need to name the `vector<core::TypePtr>` part of the initializer, or does it work without it?

Reply:
yeah, i think it has to do with explicit constructors. i got a compile error when i omitted this. i was going to play around with it.

Reply:
no, adding `explicit` to the TupleType constructor doesn't do it. it must be because `TupleType::build` is a normal method, but `make_type` is templated and forwards args?